src/CoreBundle/Util/RoutingUtilDataAccess.php

```diff
@@ -177,7 +175,6 @@ public function getAllPageAssignments(TdbCmsPortal $portal, TdbCmsLanguage $lang
           AND tn.$activeFieldName = '1'
           AND tn.`start_date` <= :date
           AND (tn.`end_date` >= :date OR tn.`end_date` = '0000-00-00 00:00:00')
-          AND p.$nameFieldName <> ''
```

Comment:
Looks reasonable: getAllPageAssignments($portal) should/could report all pages - regardless of names.